pages/common/cppcheck.md

```diff
@@ -0,0 +1,32 @@
+# cppcheck
+
+> A static analysis tool for C/C++ code.
+> Instead of syntax errors, it focuses on the types of bugs that compilers normally do not detect.
+
+- Recursively check the current folder, showing progress on the screen and logging error messages to a file:
+
+`cppcheck . 2> cppcheck.log`
+
+- Recursively check a given folder, and don't print progress messages:
+
+`cppcheck --quiet {{path/to/folder}}`
+
+- Check a given file, with all tests enabled (by default only errors are shown) except two:
+
+`cppcheck --enable=all --suppress={{check_id1}} {{path/to/file.cpp}}`
```

Comment:
Huh. I meant to add two, that's a mistake. And good point on the test_id1, I was using "checks" when I started working on this, as that's the term they use in their docuentation, and did a find/replace when I decided to switch to "test" to avoid ambiguity. Somehow I missed this one.